/**:  
  ros__parameters:
    rgbcam/x : 2.3
    rgbcam/y : 0.0
    rgbcam/z : 3.5
    rgbcam/pitch : -15.0 
    rgbcam/yaw : 0.0
    rgbcam/roll : 0.0
    rgbcam/sensor_tick : "0.033333f"
    rgbcam/image_size_x : "640"
    rgbcam/image_size_y : "480"
    rgbcam/fov : "90.0f"
    rgbcam_topic_name : "front_camera"

/**:  
  ros__parameters:
    radar/x : 2.3
    radar/y : 0.0
    radar/z : 1.5
    radar/pitch : 0.0 
    radar/yaw : 0.0
    radar/roll : 0.0
    radar/sensor_tick : "0.000001f"
    radar/horizontal_fov : "40.0f"
    radar/vertical_fov : "4.4f"
    radar/points_per_second : "50000"
    radar/range : "60.0f"
    radar_topic_name: "front_radar"
    left_radar/x : 1.9
    left_radar/y : -1.4
    left_radar/z : 0.7
    left_radar/pitch : 0.0 
    left_radar/yaw : -125.0
    left_radar/roll : 0.0
    left_radar/sensor_tick : "0.000001f"
    left_radar/horizontal_fov : "110.0f"
    left_radar/vertical_fov : "4.4f"
    left_radar/points_per_second : "50000"
    left_radar/range : "13.0f"
    left_radar_topic_name: "left_radar"
    right_radar/x : 1.9
    right_radar/y : 1.4
    right_radar/z : 0.7
    right_radar/pitch : 0.0 
    right_radar/yaw : 125.0
    right_radar/roll : 0.0
    right_radar/sensor_tick : "0.000001f"
    right_radar/horizontal_fov : "110.0f"
    right_radar/vertical_fov : "4.4f"
    right_radar/points_per_second : "50000"
    right_radar/range : "13.0f"
    right_radar_topic_name: "right_radar"

/**:  
  ros__parameters: 
    lidar/x : 2.3
    lidar/y : 0.0
    lidar/z : 1.5
    lidar/pitch : 0.0 
    lidar/yaw : 0.0
    lidar/roll : 0.0
    lidar/sensor_tick : "0.05f"
    lidar/rotation_frequency : "200.0f"
    lidar/horizontal_fov : "30.0f"
    lidar/upper_fov : "0.0f"
    lidar/lower_fov : "0.0f"
    lidar/points_per_second : "90000"
    lidar/range : "100.0f"
    lidar_topic_name : "front_lidar"

/**:  
  ros__parameters: 
    info_topic_name : "velocity"
    steer_topic_name : "steer_control"
    velocity_topic_name : "velocity_control"
